Day-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Display.css';
+
+function CalculatorDisplay(props){
+   return <div className="display"><h3>{props.currentValue}</h3></div>
```

Comment:
It's best to separate tags out onto different lines for readability. For example:
```javascript
    return (
      <div className="display">
        <h3>{props.currentValue}</h3>
      </div>
    );
```